CODEOWNERS

```diff
@@ -24,3 +24,13 @@
 
 /src/Cli/dotnet/commands/dotnet-new @DonJoseLuis
 /src/Tests/dotnet-new.Tests @DonJoseLuis
+
+# ILLink and ReadyToRun targets and tasks owned by runtime team
+/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets @agocke
+/src/Tasks/Microsoft.NET.Build.Tasks/PrepareForReadyToRunCompilation.cs @agocke
+/src/Tasks/Microsoft.NET.Build.Tasks/RunReadyToRunCompiler.cs @agocke
+/src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs @agocke
+/src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishReadyToRun.cs @agocke
+/src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAProjectWithAllFeatures.cs @agocke
```

Comment:
I mentioned this in the email - this file shouldn't be owned by the runtime team. This is a general "test publish" test that was written in .NET Core 1.0 - before we had ILLink and ReadyToRun targets.